### PR TITLE
Feature/add to do

### PR DESCRIPTION
**Pull request by Jaranag**
· Added functions to add new tasks to projects
· Added functions to app.tsx to send live projects array
· Created the 3 toDoColumns
· Mapping task.name to test if working

### CODE REVIEW

**Review by rekloyd — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -5,14 +5,9 @@ import { Project, ToDoTask } from "./types";
...
 
-
 function App() {
-  const [selectedProject, setSelectedProject] = useState<Project | null>(null);
-
-  const handleProjectSelect = (project: Project) => {
-    setSelectedProject(project);
-  };
 
+  const [selectedProjectId, setSelectedProjectId] = useState<string | null>(null);
```
Comment (SaraVictorFernandez) [inline src/App.tsx:10]: I love this optimization. It wasn't necessary to have the whole selected project in the state, only the id + the array is enough

File: src/App.tsx [hunk 2]
```diff
@@ -47,6 +45,40 @@ function App() {
     );
   };
 
+  const addTask = (taskName: string, status: string, id: string) => {
```
Comment (SaraVictorFernandez) [inline src/App.tsx:48]: I think it would be better to rename the variable `id` to `projectId` to show unequivocally what it's referring to

Comment (SaraVictorFernandez) [reply src/App.tsx:48]: Same with `status`-> `taskStatus`

**Review by SaraVictorFernandez — COMMENTED** [review]
great job

**Review by SaraVictorFernandez — APPROVED** [review]
lgtm

**Review by JoseAlfredoMiguel — APPROVED** [review]

**Review by JiajunQiuWu — APPROVED** [review]

**Pull request merged**